packages/adblocker/assets/update.js

```diff
@@ -29,9 +29,7 @@ const FILTER_LISTS = [
 const ASSETS_PATH = dirname(fileURLToPath(import.meta.url));
 
 async function downloadResource(resourceName) {
-  const {
```

Comment:
no need to this change, wonder why the linter does not enforce this formatting?

Reply:
maybe they're only scanning ts files